rig/place_and_route/utils.py

```diff
@@ -277,6 +277,10 @@ def build_routing_tables(routes, net_keys, omit_default_routes=True):
         If a routing tree has a terminating vertex whose route is set to None,
         that vertex is ignored.
 
+    .. note::
+        :py:func:`~.build_and_minimise_routing_tables` can be used to get
```

Comment:
"can be used to minimise routing tables produced by this function."

I'd include an example too (especially if you need to turn off default-route omission!)